app/controllers/admin/debate_outcomes_controller.rb

```diff
@@ -34,7 +34,8 @@ def fetch_debate_outcome
 
   def debate_outcome_params
     params.require(:debate_outcome).permit(
-      :debated_on, :overview, :transcript_url, :video_url, :debated
+      :debated_on, :overview, :transcript_url, :video_url, :debated,
+      :commons_image
```

Comment:
Small nitpick - probably best to leave it all on one line but that just stems from my desktop publishing experience where it was frowned upon to have a line with one word :smile: